.github/ISSUE_TEMPLATE/GITHUB_REQUEST_TEMPLATE.yaml

```diff
@@ -24,6 +24,12 @@ body:
         - Other
     validations:
       required: true
+  - type: textarea
+    attributes:
+      description: 'If yes, describe your repo type. For GitHub new repo specifically, please note we are still in the progress of publishing https://github.com/opensearch-project/.github/issues/296 and https://github.com/opensearch-project/.github/issues/304. For others, please confirm if you already have a GitHub repo in opensearch-project GitHub Org. Thanks.'
```

Comment:
```suggestion
      description: 'If yes, describe the publication target(s) your repo plans to use. For GitHub new repo specifically, please note we are still in the progress of publishing https://github.com/opensearch-project/.github/issues/296 and https://github.com/opensearch-project/.github/issues/304. For others, please confirm if you already have a GitHub repo in opensearch-project GitHub Org. Thanks.'
```

Reply:
To clarify on that, `repo type` is quite broad definition, many repos use multiple release targets (fe Maven and Cargo, ...) for which we would need to plan ahead